### PR TITLE
DOC: Fix PEP-8 issues in groupby.rst

### PR DESCRIPTION
**Pull request by FHaase**

### CODE REVIEW

File: doc/source/groupby.rst [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
 
    import numpy as np
-   np.random.seed(123456)
-   np.set_printoptions(precision=4, suppress=True)
    import pandas as pd
-   pd.options.display.max_rows = 15
-   import matplotlib
-   # matplotlib.style.use('default')
+
    import matplotlib.pyplot as plt
```
Comment (datapythonista) [inline doc/source/groupby.rst]: matplotlib is a third party, and pandas is our own library, so if you divide the imports, matplotlib should be with numpy, and pandas separate

File: doc/source/groupby.rst [hunk 2]
```diff
@@ -938,7 +937,7 @@ that is itself a series, and possibly upcast the result to a DataFrame:
 .. ipython:: python
 
     def f(x):
-      return pd.Series([ x, x**2 ], index = ['x', 'x^2'])
+        return pd.Series([x, x**2], index=['x', 'x^2'])
```
Comment (datapythonista) [inline doc/source/groupby.rst]: aren't spaces required around the `**` operator?

**Review by datapythonista — COMMENTED** [review]
Looks good, just couple of small details

File: doc/source/groupby.rst (hunk 2 above)
Comment (FHaase) [reply doc/source/groupby.rst]: Somehow it doesn't complain, I changed it anyway.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @FHaase

**Comment by gfyoung** [comment]
Thanks @FHaase !  Keep them coming!

**Pull request merged**